spec/models/order_item_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe OrderItem, type: :model do
+  before :each do
+    @order_item = FactoryGirl.create(:order_item)
+  end
+
+  it 'should belong to an order' do
+    expect(@order_item).to respond_to(:order)
+  end
+
+  it 'should belong to an item' do
+    expect(@order_item).to respond_to(:item)
+  end
+
+  it 'should be invalid without a quantity' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.